crates/biome_formatter/src/lib.rs

```diff
@@ -699,48 +699,42 @@ impl FromStr for BracketSameLine {
     }
 }
 
-#[derive(Clone, Copy, Debug, Default, Deserializable, Eq, Hash, Merge, PartialEq)]
+#[derive(Clone, Copy, Debug, Default, Eq, Hash, Deserializable, Merge, PartialEq)]
```

Comment:
:suspect:

Reply:
Actually I just moved `Expand` here and removed `ObjectWrap`, but the diff shows this 😅